src/backend/web/app.js

```diff
@@ -68,4 +68,12 @@ app.use(logger);
 // Include our router with all endpoints
 app.use('/', router);
 
+/**
+ * 404 Handler, Pass to front-end
+ * Leverage .status because adding the `404` status in redirect causes "Not Found. Redirecting to /404?search=" to display.
+ */
+app.use((req, res) => {
+  res.status(404).redirect(`/404`);
+});
+
```

Comment:
While you're doing this, can you add a general error handler at the end of this that uses our logger to log the error before sending the 500?

Reply:
`logger.error`? I had tried this earlier and couldn't access the logger method.

Reply:
It's due to how we export that logger.  You probably need `logger.logger`, which is horrible.